crates/aranya-crypto/src/misc.rs

```diff
@@ -1,4 +1,4 @@
-//! Utility routines for `userkeys` and `apq`.
+//! Utility routines for `devicekeys` and `apq`.
```

Comment:
I don't even know what `userskeys` is referring to. If that identifier doesn't exist in `aranya-crypto`, delete it from the doc comment.